cli/dcoscli/marathon/main.py

```diff
@@ -89,13 +90,13 @@
                                 (https://mesosphere.github.io/
                                 marathon/docs/rest-api.html#post-/v2/groups).
 
-    <instances>                 The number of instances to start
```

Comment:
We still have this positional parameter: `dcos marathon app start ... [<instances>]`